test/python/test_mapper.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring
+
+# Copyright 2017 IBM RESEARCH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+import unittest
+
+from qiskit import QuantumProgram
+from .common import QiskitTestCase
+
+class MapperTest(QiskitTestCase):
+    """Test the mapper."""
+
+    def setUp(self):
+        self.seed = 42
+        self.qp = QuantumProgram()
+
+    def tearDown(self):
+        pass
+
+    def test_issue_81(self):
```

Comment:
Along the same line of previous comments, I think we should have a better descriptive method name here.